h2o-algos/src/main/java/hex/api/MakeGLMModelHandler.java

```diff
@@ -28,6 +28,16 @@ public GLMModelV3 make_model(int version, MakeGLMModelV3 args){
       throw new IllegalArgumentException("missing source model " + args.model);
     String [] names = model._output.coefficientNames();
     Map<String,Double> coefs = model.coefficients();
+    if (args.beta.length != names.length) {
+      StringBuffer sbuff = new StringBuffer();
```

Comment:
StringBuilder is better than StringBuffer

same can also be accomplished with String.join, eg. : "model coefficients needed are: \n" + String.join('\n', names)

Reply:
Thanks Michal, I used String.join("/n", names).